packages/fx-core/src/plugins/resource/apiconnector/depsHandler.ts

```diff
@@ -48,6 +49,16 @@ export class DepsHandler {
     }
     if (needUpdate) {
       await fs.writeFile(localPkgPath, JSON.stringify(pkgContent, null, 4));
+      const telemetryProperties = {
+        component: this.componentType,
+        dependency: JSON.stringify(pkgConfig),
+      };
+
+      TelemetryUtils.sendEvent(
```

Comment:
can we name the event to something like `update-package`? `scaffold-installed` is not so straightforward for people to understand what's the meaning.